scripts/node/twig_functions.js

```diff
@@ -0,0 +1,41 @@
+/**
+ * Install Pattern Lab.
+ */
+const shell = require("shelljs");
+const fs = require('fs');
+const path = {
+  'drupalComposer': '../../../../vendor/drupal-pattern-lab',
+  'drupalNoComposer': './../../vendor/drupal-pattern-lab',
+  'standalone': 'vendor/drupal-pattern-lab'
+}
+
+function vendorPathInstalled() {
+  if (fs.existsSync(path.standalone) || fs.existsSync(path.drupalComposer) || fs.existsSync(path.drupalNoComposer)) {
```

Comment:
If you will always be checking for all paths, it will be more maintainable to map over them a la `Object.values(paths).map(fs.existsSync)`

Reply:
Which would then allow you to convert this function to 
```js
function vendorPathInstalled() {
  return Object.values(paths).map(fs.existsSync).includes(true)
}
```

Reply:
Which of course puts you at `O(n)` in all cases, so we could instead do
```js
function vendorPathInstalled() {
  return Object.values(paths).some(fs.existsSync)
}
```
which will retain the efficiency of your code while making it briefer, eliminating a branch, and allowing for expansion

Reply:
If we wanted to make this slightly less imperative, `paths` could be an argument to the function